docs/website/docs/dlt-ecosystem/verified-sources/filesystem/advanced.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Advanced filesystem usage
+description: Use filesystem source as a building block
+keywords: [readers source and filesystem, files, filesystem, readers source, cloud storage]
+---
+
+The filesystem source actually provides you with the building blocks to facilitate loading data from files. This section aims to give you more information about how you can customize the filesystem source for your use case.
+
+## Standalone Filesystem Resource
+
+You can use the [standalone filesystem](../../../general-usage/resource#declare-a-standalone-resource) resource to list files in cloud storage or local filesystem. This allows you to customize file readers or manage files using [fsspec](https://filesystem-spec.readthedocs.io/en/latest/index.html).
```

Comment:
You can use the [standalone filesystem](../../../general-usage/resource#declare-a-standalone-resource) resource to list files in cloud storage or **a** local filesystem. This allows you to customize file readers or manage files using [fsspec](https://filesystem-spec.readthedocs.io/en/latest/index.html).